src/main/kotlin/no/nav/familie/ef/personhendelse/dødsfalloppgaver/DødsfallOppgaveService.kt

```diff
@@ -0,0 +1,29 @@
+package no.nav.familie.ef.personhendelse.dødsfalloppgaver
+
+import no.nav.familie.ef.personhendelse.handler.PersonhendelseType
+import no.nav.person.pdl.leesah.Personhendelse
+import org.springframework.stereotype.Service
+import java.time.LocalDateTime
+import java.util.UUID
+
+@Service
+class DødsfallOppgaveService(
+    private val dødsfallOppgaveRepository: DødsfallOppgaveRepository,
+) {
+    fun lagreDødsfallOppgave(personhendelse: Personhendelse, personhendelseType: PersonhendelseType, personIdent: String, beskrivelse: String) {
+        dødsfallOppgaveRepository.lagreOppgave(
+            UUID.fromString(personhendelse.hendelseId),
+            personhendelseType,
+            personhendelse.endringstype.name,
+            personIdent,
+            beskrivelse,
+            LocalDateTime.now(),
+        )
+    }
+    fun hentAlleDødsfallsOppgaver(): List<DødsfallOppgave> {
```

Comment:
Kan gjerne hete det samme som repo-metoden som blir kalt for å unngå misforståelse